src/app/core/configurable-enum/basic-autocomplete/basic-autocomplete.component.ts

```diff
@@ -142,6 +143,7 @@ export class BasicAutocompleteComponent<O, V = O>
     let filteredOptions = this._options;
     this.showAddOption = false;
     clearTimeout(this.addOptionTimeout);
+    setTimeout(() => (this.showAddOption = false), 100);
```

Comment:
observing actual users add options, I am tending towards removing the delay for new options again. It was complex enough for a non-savvy user to figure out they can do this without things moving and changing. Maybe instead highlight the "Add new" option with different color a bit more than already?

Sorry, somewhat off-topic here actually. Just popped into my mind.

Reply:
Got point. I removed the timeout and instead visually highlighted the add option text. Please have a look

Reply:
Nice. I like it.

In terms of UI logic, maybe we should use the blue accent color though (similar to add buttons above lists, etc.)?

Reply:
I get the point but it does not look good with the blue color.